plugins/php/php_plugin.c

```diff
@@ -6,6 +6,14 @@ static sapi_module_struct uwsgi_sapi_module;
 
 static int uwsgi_php_init(void);
 
+#ifdef UWSGI_PHP7
+typedef ssize_t php_strlen_size;
```

Comment:
Are you sure you want ssize_t and not size_t?

Reply:
Aigh, good catch... this is actually a typo... wrote it correctly in the commit message but didn't in the actual code. Thanks, will fix it ASAP.